server/src/main/scala/io/delta/kernelsharedtable/DeltaSharedTableKernel.scala

```diff
@@ -138,7 +149,6 @@ class DeltaSharedTableKernel(
           case e: io.delta.kernel.exceptions.TableNotFoundException =>
             throw new FileNotFoundException(e.getMessage)
         }
-
```

Comment:
nit: is this and below intended?

Reply:
yeah wanted to clean up some unneeded whitespace. should i bring it back?

Reply:
clean up is preferred.